### PR TITLE
Add extensibility point to load tab icons from another source

### PR DESCRIPTION
**Pull request by softlion**
### Description of Change

Added an extensibily point to be able to load tab icons directly from a custom Drawable. 
This is the only way to get them to load from embedded svg files.
### API Changes

Added a protected method to TabbedPageRenderer :
- SetTabIcon(TabLayout.Tab tab, FileImageSource icon)
### Behavioral Changes

None
### PR Checklist
- [ ] Has tests (if omitted, state reason in description)
- [X] Rebased on top of master at time of PR
- [X] Changes adhere to coding standard
- [X] Consolidate commits as makes sense

There is no test project for the Android platform project.
The change is extremely simple, but powerful.

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs [hunk 1]
```diff
@@ -331,11 +331,16 @@ void UpdateTabIcons()
...
 
 				TabLayout.Tab tab = tabs.GetTabAt(i);
-				tab.SetIcon(ResourceManager.IdFromTitle(icon, ResourceManager.DrawableClass));
+			    SetTabIcon(tab, icon);
 			}
 		}
 
-		void UpdateBarBackgroundColor()
+	    protected virtual void SetTabIcon(TabLayout.Tab tab, FileImageSource icon)
+	    {
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs]: the indentation seems to be wrong, can you please fix the style.

File: Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs [hunk 2]
```diff
@@ -331,11 +331,16 @@ void UpdateTabIcons()
...
+            tab.SetIcon(ResourceManager.IdFromTitle(icon, ResourceManager.DrawableClass));
+        }
+
+        void UpdateBarBackgroundColor()
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs]: also this shouldn't be need

Comment (softlion) [inline Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs]: you are right, don't know why git changed this. What is the correct setting for git: update cr/lf to cr only ? replace space with tabs ?

**Comment by softlion** [comment]
I've created the same extesibility point for iOS. PR is near.

**Comment by StephaneDelcroix** [comment]
I fixed the indentation and manually merged this branch. Take this as a favor, @softlion, not an habit

**Comment by softlion** [comment]
Yes i know :)
But i don't know why it messed with the spaces and tabs.
I'm using only spaces explicitly, no reformat options, and have no issues with other git repos.
Sorry for this.

**Comment by StephaneDelcroix** [comment]
> I'm using only spaces explicitly, no reformat options, and have no issues with other git repos.

We use tabs

**Pull request closed without merging**